src/util/memory.rs

```diff
@@ -157,10 +174,19 @@ impl std::fmt::Display for MmapAnnotation<'_> {
 /// Check the result from an mmap function in this module.
 /// Return true if the mmap has failed due to an existing conflicting mapping.
 pub(crate) fn result_is_mapped(result: Result<()>) -> bool {
+    #[cfg(not(target_os = "windows"))]
     match result {
         Ok(_) => false,
         Err(err) => err.raw_os_error().unwrap() == libc::EEXIST,
     }
+    #[cfg(target_os = "windows")]
+    match result {
+        Ok(_) => false,
+        Err(err) => {
+            // ERROR_INVALID_ADDRESS may be returned if the address is already mapped or invalid
+            err.raw_os_error().unwrap() == 487 // ERROR_INVALID_ADDRESS
```

Comment:
Use the constant: https://docs.rs/windows-sys/0.61.2/windows_sys/Win32/Foundation/constant.ERROR_INVALID_ADDRESS.html